extensions/amp-animation/0.1/runners/animation-runner.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * Copyright 2019 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  WebAnimationPlayState, // eslint-disable-line no-unused-vars
+} from '../web-animation-types';
+
+/**
+ */
+export class AnimationRunner {
+
+  /**
+   * @param {!Array<!../web-animation-types.InternalWebAnimationRequestDef>} requests
+   */
+  constructor(requests) {
+    /** @const @protected */
```

Comment:
nit: What's the purpose of `protected` here? I believe we only need this for module-level scoping, which this doesn't require.